main/lsp/requests/completion.cc

```diff
@@ -849,89 +850,105 @@ vector<unique_ptr<CompletionItem>> CompletionTask::getCompletionItems(LSPTypeche
 
     // ----- methods -----
```

Comment:
Do we want a timer for the `// ----- keywords -----` stuff above this?

Reply:
Good question.  I thought about adding it, and then decided that this was pretty static: 30 or so prefix matches on a table of keywords *should* be pretty quick and not worth measuring?

Reply:
Works for me!